pages/app/components/page-view.tsx

```diff
@@ -15,7 +17,9 @@ export default function PageView({ pageId }: { pageId: string }) {
   return (
     <ErrorBoundary key={pageId}>
       <Suspense fallback={<span>Loading...</span>}>
-        <Page />
+        <div className={styles['page-container']}>
```

Comment:
Do we really need this wrapper? Can we apply the same style to the `<div id="app">` that we have already?

Reply:
Great suggestion, applied the change 👍